web/blog/tenzir-v4.16/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Tenzir v4.16
+authors: [dominiklohmann]
+date: 2024-06-05
+tags: [release, context, rpm]
+comments: true
+---
+
+Pipelines now connect more flexibly than ever before with [Tenzir
+v4.16](https://github.com/tenzir/tenzir/releases/tag/v4.16.0) and its upgraded
+[`publish`](/operators/publish) and [`subscribe`](/operators/subscribe)
+operators.
+
+![Tenzir v4.16](tenzir-v4.16.excalidraw.svg)
+
+<!-- truncate -->
+
+## Multi-Producer Multi-Consumer
+
+The introduction of the `publish` operator with [Tenzir
+v4.12](/blog/tenzir-v4.12) enabled split-routing of events. We frequently saw
+users write pipelines like this:
+
+```text {0} title="Pipeine 1: Publish alerts"
+…
+| publish alerts
+```
+
+```text {0} title="Pipeine 2: Save alerts with a high risk score to Splunk"
+subscribe alerts
+| where risk_score >= 90
+| fluent-bit splunk …
+```
+
+```text {0} title="Pipeine 3: Save all alerts to S3 for later reference"
+subscribe alerts
+| write json --compact-output
+| compress zstd
```

Comment:
```suggestion
| compress zstd
```

Isn't this implied by the `.zst` extension?

Reply:
I simplified it to use the short-form.